### PR TITLE
S5753: Add ITs for csharp and vb.net

### LINKED ISSUE #3185: Rule S5753: Disabling ASP.NET "Request Validation" feature is security-sensitive
Implements https://jira.sonarsource.com/browse/RSPEC-5753

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**
Fixes #3185

### CODE REVIEW

File: its/projects/WebConfig.CSharp/Web.config [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
```
Comment (andrei-epure-sonarsource) [inline its/projects/WebConfig.CSharp/Web.config:2]: as a note, this has more contents than the VB equivalent? is it intended?

File: its/src/test/java/com/sonar/it/csharp/WebConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  @Test
+  public void should_raise_hotspot_on_web_config() throws Exception {
+    final String projectName = "WebConfig.CSharp";
+
+    Tests.analyzeProject(temp, projectName, null);
+
+    List<Hotspot> hotspots = Tests.getHotspots(projectName);
+    assertThat(hotspots.size()).isEqualTo(2);
+
+    assertHotspot(hotspots.get(0), 16);
```
Comment (andrei-epure-sonarsource) [inline its/src/test/java/com/sonar/it/csharp/WebConfigTest.java:49]: could you also check that it's raised on the `web.config` file?

File: its/projects/WebConfig.CSharp/Web.config (hunk 1 above)
Comment (costin-zaharia-sonarsource) [reply its/projects/WebConfig.CSharp/Web.config:2]: Not necessary. I will remove what is not needed.

File: its/src/test/java/com/sonar/it/vbnet/WebConfigTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    TestUtils.reset(ORCHESTRATOR);
+  }
+
+  @Test
+  public void should_raise_hotspot_on_web_config() throws Exception {
+    final String projectName = "WebConfig.Vb";
+
+    Tests.analyzeProject(temp, projectName, null);
+
+    List<Hotspots.SearchWsResponse.Hotspot> hotspots = Tests.getHotspots(projectName);
```
Comment (andrei-epure-sonarsource) [inline its/src/test/java/com/sonar/it/vbnet/WebConfigTest.java:46]: same suggestion here, to get hotspots for the file name

**Review by andrei-epure-sonarsource — CHANGES_REQUESTED** [review]

File: its/src/test/java/com/sonar/it/csharp/WebConfigTest.java (hunk 2 above)
Comment (costin-zaharia-sonarsource) [reply its/src/test/java/com/sonar/it/csharp/WebConfigTest.java:49]: You mean by checking the component ("WebConfig:Web.config")? I'm not sure we have any other way.

Comment (andrei-epure-sonarsource) [reply its/src/test/java/com/sonar/it/csharp/WebConfigTest.java:49]: yes. From reading the test, it's not obvious that the security hotspots are in the Web.config file

**Review by andrei-epure-sonarsource — APPROVED** [review]
LGTM. I will merge to the feature branch as discussed, to have it red.

**Pull request merged**